README.md

```diff
@@ -51,12 +51,7 @@ See [Troubleshooting](docs/troubleshooting.md) for common pitfalls and remediati
 
 If you're still encountering a non-critical issues, please contact clrieowners@microsoft.com.
 
-Critical security issues and bugs should be reported privately, via email, to the Microsoft Security
-Response Center (MSRC) at [secure@microsoft.com](mailto:secure@microsoft.com). You should
-receive a response within 24 hours. If for some reason you do not, please follow up via
-email to ensure we received your original message. Further information, including the
-[MSRC PGP](https://technet.microsoft.com/security/dn606155) key, can be found in
-the [Security TechCenter](https://technet.microsoft.com/security/default).
+Critical security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) at [secure@microsoft.com](mailto:secure@microsoft.com. See [Security](SECURITY.md) for more details.
```

Comment:
I think you are missing a ")" between the "secure@microsoft.com" and the ending "." of the sentence.

Reply:
I don't see the file SECURITY.md in this commit.

Reply:
Thanks